bin/pysparkling2.cmd

```diff
@@ -11,12 +11,9 @@ rem end of checking Sparkling environment
 call %LIBSW% :banner
 
 rem setup pysparkling command line
-rem Because of SPARK-18648 we need to put assembly also on driver/executor class paths
-SET SPARK_OPT_JARS=--jars !FAT_JAR_FILE! --conf spark.driver.extraClassPath=!FAT_JAR_FILE! --conf spark.executor.extraClassPath=!FAT_JAR_FILE!
 SET SPARK_OPT_PYFILES=--py-files !PY_ZIP_FILE!
-cd %TOPDIR%
 SET PYTHONPATH=%PY_ZIP_FILE%:%PYTHONPATH%
-call !SPARK_HOME!/bin/pyspark2.cmd !SPARK_OPT_PYFILES! %SPARK_OPT_JARS% %*
```

Comment:
I will remove this part of the change if you think it's dangerous.

I will keep the rest because then it will be consistent with shell scripts where we don't need to handle deploy mode explicitly as well